### PR TITLE
Added a delete unverified user from the table after every 1 minute

### PR DESCRIPTION
**Pull request by BlessingAmoah**
I added an option to delete a user after every 1 minute if the user is not verified.

### CODE REVIEW

File: backend/routes/auth.js [hunk 1]
```diff
@@ -158,6 +160,15 @@ router.post('/resendverification', async (req, res) => {
   }
 });
 
+// Delete unverified users every minute
+schedule.scheduleJob('*/1 * * * *', async () => {
+  await User.destroy({
+    where: {
+      emailVerified: false,
+      createdAt: { [Op.lt]: new Date(Date.now() - 60000) }
```
Comment (Mustafa-Sadriwala) [inline backend/routes/auth.js:168]: can we actually do like 10 minutes? Or will that be too hard to test? Now that we know it works don't think 1 minutes is sufficient for real users

**Comment by brittanylau** [comment]
the code syntactically looks ok to me, but was hoping to understand why we're making this change! I noticed in your project plan you mentioned users should request a new verification if it expires - how can they do that if the user is deleted?

File: backend/routes/auth.js (hunk 1 above)
Comment (BlessingAmoah) [reply backend/routes/auth.js]: Yes, I have changed the time to 10 minutes.

**Comment by BlessingAmoah** [comment]
> the code syntactically looks ok to me, but was hoping to understand why we're making this change! I noticed in your project plan you mentioned users should request a new verification if it expires - how can they do that if the user is deleted?

Yes, that was the initial plan but if the user is not deleted from the database his or her information is stored. It is not possible to make a new account due to the user details already in the database. So deleting it will make it possible for the user to create a new account.

**Pull request merged**